packages/calendar/src/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import {useState} from 'react'
+import getUserLocale from 'get-user-locale'
+import {useDefaults} from '@smashing/theme'
+import {CalendarProps, CalendarAppearanceType} from './types'
+import {StyledContainer, StyledInput, StyledCalendar} from './styles'
+
+const Calendar: React.FC<CalendarProps> = ({
+  onChange = (date: Date) => {},
+  value,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false)
```

Comment:
It's good practice to add `is/has` prefix to boolean variables:

```suggestion
  const [isOpen, setIsOpen] = useState(false)
```